dkp-toolchain-vars/cubevars.sh

```diff
@@ -6,7 +6,7 @@ export PORTLIBS_CUBE=${PORTLIBS_PREFIX}
 
 export CFLAGS="-O2 -mogc -mcpu=750 -meabi -mhard-float -ffunction-sections -fdata-sections"
 export CXXFLAGS="${CFLAGS}"
-export CPPFLAGS="-D__GAMECUBE__ -I${PORTLIBS_CUBE}/include -I${PORTLIBS_PPC}/include"
+export CPPFLAGS="-D__GAMECUBE__ -I$DEVKITPRO/libogc/include -I${PORTLIBS_CUBE}/include -I${PORTLIBS_PPC}/include"
```

Comment:
${DEVKITPRO} for consistency with rest of file.

Reply:
Done!